src/RichTextEditor/index.js

```diff
@@ -58,6 +58,19 @@ const RichTextEditor = (props) => {
   const { isEditable, canBeFormatted } = props;
 
   const onKeyDown = useCallback((event) => {
+    if (isHotkey(ENTER, event)) {
+      // eslint-disable-next-line no-restricted-syntax
+      for (const [n] of Node.ancestors(editor, editor.selection.focus.path, { reverse: true })) {
```

Comment:
To add to our discussion we had over zoom - I think we don't want to pass the reverse option so that we go bottom up as the docs say: "By default, the order is bottom-up, from lowest to highest ancestor in the tree."
![Screen Shot 2020-04-14 at 4 41 57 PM](https://user-images.githubusercontent.com/20543103/79272064-e2839d00-7e6e-11ea-9748-8a2032ec8a2b.png)

Reply:
When I pass reverse in, I get this log order:
1. paragraph
2. block quote
3. editor

When I don't pass reverse in, I get this log order:
1. editor
2. block quote
3. paragraph

Reply:
Could be semantics and they are saying the editor is at the bottom?

Reply:
wow, that is confusing!